aci/data_source_aci_vnsabsnode.go

```diff
@@ -97,6 +97,22 @@ func dataSourceAciFunctionNode() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"l4_l7_device_interface_consumer_connector_type": &schema.Schema{
```

Comment:
Did you forget to include the function node doc update?

Reply:
yes, added